config/plugins.yaml

```diff
@@ -848,6 +858,30 @@ plugins:
     - verify-owners
     - welcome
     - wip
+  falcosecurity/plugin-sdk-cpp:
+    plugins:
+    - approve
+    - assign
+    - blunderbuss
+    - branchcleaner
+    - cat
+    - dco
+    - dog
+    - goose
+    - help
+    - hold
+    - label
+    - lifecycle
+    - lgtm
+    - mergecommitblocker
+    - release-note
```

Comment:
```suggestion
```
We probably do not want the release note block, since the repo is not so big and it would be cumbersome.